internal/archive/zip.go

```diff
@@ -0,0 +1,49 @@
+package archive
+
+import (
+	"archive/zip"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+// Zip will compress and zip up all payloads for the test project
+func Zip(src, target string) error {
+	output, err := os.Create(target)
+	if err != nil {
+		return err
+	}
+	defer output.Close()
+
+	w := zip.NewWriter(output)
+	addFiles(w, src, "")
+	return w.Close()
+}
+func addFiles(w *zip.Writer, basePath, baseInZip string) error {
+	files, err := ioutil.ReadDir(basePath)
+	if err != nil {
+		return err
+	}
+	for _, file := range files {
+		if !file.IsDir() {
+			data, err := ioutil.ReadFile(basePath + file.Name())
```

Comment:
This is too brittle. Don't concat file paths on your own unless there is a good reason to do so:
```suggestion
			data, err := ioutil.ReadFile(filepath.Join(basePath, file.Name()))
```

Reply:
OK. I'll update that.